src/components/HeaderLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { NavLink, useLocation } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+interface Props {
+  title: string
+  path: string
+}
+
+export default function HeaderLink({ title, path }: Props) {
+  const isActive = path === useLocation().pathname
+
+  return (
+    <Link exact to={path} as={isActive ? Heading : undefined}>
```

Comment:
Pretty! 🚀

Reply:
Thanks for pointing me in the right direction 😉